x-pack/plugins/ml/public/alerting/jobs_health_rule/register_jobs_health_alerting_rule.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { lazy } from 'react';
+import { i18n } from '@kbn/i18n';
+import { TriggersAndActionsUIPublicPluginSetup } from '../../../../triggers_actions_ui/public';
+import { PluginSetupContract as AlertingSetup } from '../../../../alerting/public';
+import { ML_ALERT_TYPES } from '../../../common/constants/alerts';
+import { MlAnomalyDetectionJobsHealthRuleParams } from '../../../common/types/alerts';
+
+export function registerJobsHealthAlertingRule(
+  triggersActionsUi: TriggersAndActionsUIPublicPluginSetup,
+  alerting?: AlertingSetup
+) {
+  triggersActionsUi.alertTypeRegistry.register({
+    id: ML_ALERT_TYPES.AD_JOBS_HEALTH,
+    description: i18n.translate('xpack.ml.alertTypes.jobsHealthAlertingRule.description', {
+      defaultMessage: 'Alert when anomaly detection jobs experiencing realtime issues.',
+    }),
+    iconClass: 'bell',
+    documentationUrl(docLinks) {
+      return `${docLinks.ELASTIC_WEBSITE_URL}guide/en/machine-learning/${docLinks.DOC_LINK_VERSION}/ml-configuring-alerts.html`;
+    },
+    alertParamsExpression: lazy(() => import('./anomaly_detection_jobs_health_rule_trigger')),
+    validate: (alertParams: MlAnomalyDetectionJobsHealthRuleParams) => {
+      const validationResult = {
+        errors: {
+          includeJobs: new Array<string>(),
+          testsConfig: new Array<string>(),
+        } as Record<keyof MlAnomalyDetectionJobsHealthRuleParams, string[]>,
+      };
+
+      if (!alertParams.includeJobs?.jobIds?.length && !alertParams.includeJobs?.groupIds?.length) {
+        validationResult.errors.includeJobs.push(
+          i18n.translate('xpack.ml.alertTypes.jobsHealthAlertingRule.includeJobs.errorMessage', {
+            defaultMessage: 'Job selection is required',
```

Comment:
It seems like this message appears twice: 
![image](https://user-images.githubusercontent.com/26471269/126397572-45dd395b-1925-49c8-8e08-fc4f2cbe6828.png)
If it's possible to avoid duplicating it, my preference is for the less obtrusive message directly under the problematic field.

Reply:
I've used [EuiFieldSet](https://elastic.github.io/eui/#/forms/form-controls#fieldset-and-legend) for health checks selection and it doesn't allow appending an error, that's why I added form error messages on top. "Please address the highlighted errors" comes from the [EUI validation](https://elastic.github.io/eui/#/forms/form-validation) but as you suggested I'll hide it and add an error message below the fieldset

Reply:
Changed in https://github.com/elastic/kibana/pull/106084/commits/2ada9eb3ee24ed1cc79d857a507a3eb5349573bb